app/bundles/LeadBundle/Tests/EventListener/CampaignSubscriberFunctionalTest.php

```diff
@@ -90,59 +92,21 @@ protected function beforeBeginTransaction(): void
 
     public function testUpdateLeadAction(): void
     {
-        $application = new Application(self::$kernel);
-        $application->setAutoExit(false);
-        $applicationTester = new ApplicationTester($application);
-
         $contactIds = $this->createContacts();
         $campaign   = $this->createCampaign($contactIds);
 
         // Force Doctrine to re-fetch the entities otherwise the campaign won't know about any events.
         $this->em->clear();
 
         // Execute the campaign.
-        $exitCode = $applicationTester->run(
-            [
-                'command'       => 'mautic:campaigns:trigger',
-                '--campaign-id' => $campaign->getId(),
-            ]
-        );
-
-        Assert::assertSame(0, $exitCode, $applicationTester->getDisplay());
-
-        /** @var Lead $contactA */
-        $contactA = $this->contactRepository->getEntity($contactIds[0]);
-        /** @var Lead $contactB */
-        $contactB = $this->contactRepository->getEntity($contactIds[1]);
-        /** @var Lead $contactC */
-        $contactC = $this->contactRepository->getEntity($contactIds[2]);
-
-        $this->assertEquals(42, $contactA->getPoints());
-        $this->assertEquals(42, $contactB->getPoints());
-        $this->assertEquals(42, $contactC->getPoints());
-    }
-
-    public function testLeadFieldValueDecisionWithUTM(): void
-    {
-        $application = new Application(self::$kernel);
-        $application->setAutoExit(false);
-        $applicationTester = new ApplicationTester($application);
-
-        $contactIds = $this->createContacts();
-        $campaign   = $this->createCampaign($contactIds);
```

Comment:
@aarohiprasad looking at the 2 PRs that were pushed in this one I don't see those removing such a chunk of code. Can you please review why these 2 tests were merged into 1?

Reply:
@escopecz there were 3 PRs pushed in this one. Check the last one out :)
https://github.com/acquia/mc-cs/pull/1595/files#diff-5abd01cd0b14c3c22ceb6accf2a3531fb315fbc0723df193c8bf925137126fa1